zerver/lib/rate_limiter.py

```diff
@@ -67,15 +90,17 @@ def remove_ratelimit_rule(range_seconds, num_requests):
 def block_user(user, seconds, domain='all'):
     # type: (UserProfile, int, Text) -> None
     "Manually blocks a user id for the desired number of seconds"
-    _, _, blocking_key = redis_key(user, domain)
+    entity = RateLimitedUser(user, domain=domain)
+    _, _, blocking_key = list(entity.keys())
```

Comment:
I think linter mistook `keys` for the `keys` function of dictionary. I had to change the name to fix it.